components/admin_console/elasticsearch_settings.jsx

```diff
@@ -312,7 +315,7 @@ export default class ElasticsearchSettings extends AdminSettings {
                                 createJobHelpText={
                                     <FormattedMessage
                                         id='admin.elasticsearch.createJob.help'
-                                        defaultMessage='All posts in the database will be indexed from oldest to newest. Elasticsearch is available during indexing but search results may be incomplete until the indexing job is complete.'
+                                        defaultMessage='All users, channels and posts in the database will be indexed from oldest to newest. Elasticsearch is available during indexing but search results may be incomplete until the indexing job is complete.'
```

Comment:
I guess this text depends on the `enableAutocomplete` setting. Another question, can I have enabled autocomplete but not enabled searching?

Reply:
The bulk index is independent of your search settings, so it will always index the three entities.

> Another question, can I have enabled autocomplete but not enabled searching?

Yes, they are independent flags in the system console